### ISSUE TITLE
List incorrect rendering - count

### ISSUE THREAD

**Issue by mrpvb-pvbk** [asjqkkkk/markdown_widget#28]
**Describe the bug**
I use this data:
```
1. **App Name** - This is the name of the app.
2. **Organization identifier** - This is the unique name used together with the app name to uniquely identify your app on the Google Play Store.
3. **Target platforms** - This is the platforms on which the app can run, its always advisable to choose the version of the Android that can support at least 90% of the devices in the wild. 
4. **Theme** - The default theme that will be applied to the entire app. This can be changed and configured later.
```
What should it look like:
![image](https://user-images.githubusercontent.com/44783405/83045583-6f18a200-a04e-11ea-83c9-a04d5ff104a2.png)
Widget:
![image](https://user-images.githubusercontent.com/44783405/83045707-a0916d80-a04e-11ea-8af9-6ccbfa7bd79c.png)
The error is that the invoice starts from zero.

**Comment by asjqkkkk** [comment]
This feature is currently not available on the web，see the reason #2 

in mobile you can set `selectable` to false in `PConfig` to solve this problem

**Comment by mrpvb-pvbk** [comment]
But i need to use `selectable ` too

**Comment by asjqkkkk** [comment]
> But i need to use `selectable ` too

flutter do not support insert `WidgetSpan` in `SelectableText.rich` , so there is not a good way to solve this problem😅

**Comment by mrpvb-pvbk** [comment]
What about set `selectable `only for this element?

**Comment by asjqkkkk** [comment]
> What about set `selectable `only for this element?

It's not a bad idea, but it seems strange that some of them can't be copied

I'll add `selectable` to `OlConfig` and `UlConfig`, or you can submit a pr to fix it😄

**Comment by mrpvb-pvbk** [comment]
> > What about set `selectable `only for this element?
> 
> It's not a bad idea, but it seems strange that some of them can't be copied
> 
> I'll add `selectable` to `OlConfig` and `UlConfig`, or you can submit a pr to fix it😄

Nice👌 I hope it will works in the web too😅

**Comment by asjqkkkk** [comment]
> > > What about set `selectable `only for this element?
> > 
> > 
> > It's not a bad idea, but it seems strange that some of them can't be copied
> > I'll add `selectable` to `OlConfig` and `UlConfig`, or you can submit a pr to fix it😄
> 
> Nice👌 I hope it will works in the web too😅

I hope so， but I need wait for flutter_web to solve this [issue](https://github.com/flutter/flutter/issues/42086#issuecomment-631316229)

**Comment by mrpvb-pvbk** [comment]
> This feature is currently not available on the web，see the reason #2
> 
> in mobile you can set `selectable` to false in `PConfig` to solve this problem

Strange, I did so, but the counter again with 0.

**Comment by asjqkkkk** [comment]
> > This feature is currently not available on the web，see the reason #2
> > in mobile you can set `selectable` to false in `PConfig` to solve this problem
> 
> Strange, I did so, but the counter again with 0.

maybe this is a mistake😂，I'll fix it later.
you can also change it by yourself with `OlConfig `'s `IndexWidget`
